doc/dash/dash-sonic-hld.md

```diff
@@ -1006,7 +1006,8 @@ Refer DASH documentation for the test plan.
     {
         "DASH_VNET_TABLE:Vnet1": {
             "vni": "45654",
-            "guid": "559c6ce8-26ab-4193-b946-ccc6e8f930b2"
+            "guid": "559c6ce8-26ab-4193-b946-ccc6e8f930b2",
+            "peer_list": ["Vnet200"]
```

Comment:
I don't think peer_list here matters. This was added initially but no longer used. Peering vnet can be just mentioned in the Route table

Reply:
Makes sense. The DASH_ROUTE_GROUP_TABLE allows to effective link via routing peered vnets
Will adjust this and remove all references to peer_list in the doc